Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/CaseService.java

```diff
@@ -80,4 +82,18 @@ public CaseDTO updateCase(long suitId, CaseDTO cs) {
 
         return cs;
     }
+
+    public void removeCases(long suitId, List<CaseDTO> casesToRemove) {
```

Comment:
Maybe we should have something like 'removeAllCases' method and pass there just a suitId?

Reply:
We don't delete all cases from suit, we delete only checked cases, so we need to pass suitId and the list of cases id.

Reply:
I see. Maybe it would be better to pass a list of case ids then (we are already using a suitId here)?